src/modules/translation/languages/zh.json

```diff
@@ -1396,7 +1396,11 @@
     "add_item": "新增项目",
     "limit_reached": "每笔交易只能铸造 {max} 件物品",
     "no_items": "您没有可用于\"{name}\"的项目",
-    "unsynced_warning": "有些项目有尚未批准的更改。"
+    "unsynced_warning": "有些项目有尚未批准的更改。",
+    "next": "下一個",
+    "total_items": "要鑄造的總物品",
+    "confirm_title": "您將要鑄造 {count} {count, plural, 一 {Asset} other {Assets}} 件物品，每件物品都將以提供的數量鑄造到給定的地址。",
```

Comment:
It seems that in Chinese there's no need to use the pluralized form.
If you translate:
```
You are about to mint 1 item, each item will be minted to the given address with the provided amount.
```
and replace 1 to {count}, it should work fine.